ArchiSteamFarm/Bot.cs

```diff
@@ -439,16 +439,34 @@ internal sealed class Bot {
 		}
 
 		internal static string ResponseStatus(string botName) {
+			Bot bot;
+			string result="";
```

Comment:
No need to declare result as string, because we return only one string always, instead, just fall back to returns as initially coded.